crates/rolldown/tests/esbuild/default/package_alias_match_longest/artifacts.snap

```diff
@@ -3,22 +3,40 @@ source: crates/rolldown_testing/src/integration_test.rs
 ---
 # warnings
 
-## UNRESOLVED_IMPORT
+## DIAGNOSABLE_RESOLVE_ERROR
 
 ```text
-[UNRESOLVED_IMPORT] Warning: "pkg/bar/baz" is imported by "entry.js", but could not be resolved – treating it as an external dependency.
+[DIAGNOSABLE_RESOLVE_ERROR] Warning: Could not resolve "pkg/bar/baz" in entry.js
+   ╭─[entry.js:5:8]
+   │
+ 5 │ import "pkg/bar/baz"
+   │        ──────┬──────  
+   │              ╰──────── Matched alias not found
+───╯
 
 ```
-## UNRESOLVED_IMPORT
+## DIAGNOSABLE_RESOLVE_ERROR
 
 ```text
-[UNRESOLVED_IMPORT] Warning: "pkg/baz" is imported by "entry.js", but could not be resolved – treating it as an external dependency.
+[DIAGNOSABLE_RESOLVE_ERROR] Warning: Could not resolve "pkg/baz" in entry.js
+   ╭─[entry.js:6:8]
+   │
+ 6 │ import "pkg/baz"
+   │        ────┬────  
+   │            ╰────── Matched alias not found
+───╯
 
 ```
-## UNRESOLVED_IMPORT
+## DIAGNOSABLE_RESOLVE_ERROR
 
 ```text
-[UNRESOLVED_IMPORT] Warning: "pkg/foo/bar/baz" is imported by "entry.js", but could not be resolved – treating it as an external dependency.
+[DIAGNOSABLE_RESOLVE_ERROR] Warning: Could not resolve "pkg/foo/bar/baz" in entry.js
```

Comment:
```suggestion
[UNRESOLVED_IMPORT] Warning: Could not resolve "pkg/foo/bar/baz" in entry.js
```

`UNRESOLVED_IMPORT` seems more meaningful than `DIAGNOSABLE_RESOLVE_ERROR`.

Reply:
And since we are doing `treating it as an external dependency.`, it's better to keep this message.

Reply:
I don't apply the suggestion, since it is generated by snapshot

Reply:
Also, `unresolved_import` can't give user code frame, I will refactor the resolve diagnostic after this pr.